x-pack/plugin/src/test/resources/rest-api-spec/test/ml/jobs_get_stats.yml

```diff
@@ -88,18 +89,22 @@ setup:
   - do:
       ml.get_job_stats:
         job_id: job-stats-test
-  - match: { jobs.0.job_id : job-stats-test }
-  - match: { jobs.0.data_counts.processed_record_count: 2 }
-  - match: { jobs.0.data_counts.processed_field_count: 4 }
-  - match: { jobs.0.data_counts.input_field_count: 4 }
-  - match: { jobs.0.model_size_stats.model_bytes: 0 }
-  - match: { jobs.0.state: opened }
-  - is_true: jobs.0.node.name
-  - is_true: jobs.0.node.transport_address
-  - match: { jobs.0.node.attributes.ml\.max_open_jobs: "20"}
-  - is_true: jobs.0.open_time
-  - match: { jobs.0.timing_stats.job_id: job-stats-test }
-  - gte:   { jobs.0.timing_stats.bucket_count: 0 }
+  - match:  { jobs.0.job_id : job-stats-test }
+  - match:  { jobs.0.data_counts.processed_record_count: 3 }
+  - match:  { jobs.0.data_counts.processed_field_count: 6 }
+  - match:  { jobs.0.data_counts.input_field_count: 6 }
+  - gte:    { jobs.0.model_size_stats.model_bytes: 0 }
+  - match:  { jobs.0.state: opened }
+  - is_true:  jobs.0.node.name
+  - is_true:  jobs.0.node.transport_address
+  - match:  { jobs.0.node.attributes.ml\.max_open_jobs: "20"}
+  - is_true:  jobs.0.open_time
+  - match:  { jobs.0.timing_stats.job_id: job-stats-test }
+  - match:  { jobs.0.timing_stats.bucket_count: 1 }  # Records are 1h apart and bucket span is 1h so 1 bucket is produced
+  - gt:     { jobs.0.timing_stats.minimum_bucket_processing_time_ms: 0.0 }
+  - gt:     { jobs.0.timing_stats.maximum_bucket_processing_time_ms: 0.0 }
+  - gt:     { jobs.0.timing_stats.average_bucket_processing_time_ms: 0.0 }
+  - gt:     { jobs.0.timing_stats.exponential_average_bucket_processing_time_ms: 0.0 }
```

Comment:
It's probably safer to use `gte` on lines 104-107 and 143-146.  The jobs are only creating results for 1 bucket, and if the bucket processing time for that 1 bucket happens to round down to 0 then the `gt` condition will fail.

Reply:
Done.